PROJECT_CHARTER_TEMPLATE.md

```diff
@@ -0,0 +1,77 @@
+# ${PROJECT} Charter
+
+_note: the purpose of a project charter is to provide a brief introduction_
+_to the project from a technical, community, or business perspective. The_ 
+_document also connects a project's community leadership and governance with the_
+_OpenJS Foundation's governance._
+
+
+## Section 0: Guiding Principles (optional)
+
+_directions: provide a concise, high-level statement about_
+_the project's long-term principles, values, or mission._
+
+## Section 1: Scope
+
+_directions: Include a 3-4 sentence summary of what the project does,_
+_and/or what problems it solves. Imagine trying to explain your work_
+_to a colleague who is familiar with related technical concepts but unfamiliar_
+_with the project. You may also want to describe the project's value to community_
+_and/or business stakeholders._
+
+### 1.1: In-scope (optional)
+
+_directions: list or bullet out problem spaces, use cases, repositories_
+_or other projects which are included with the work but may not be readily_
+_apparent. This may help differentiate the project from other solutions in the_
+_space._
+
+### 1.2: Out-of-Scope (optional)
+
+_directions: list or bullet out areas that may be seen to be related but are_
+_not included in the scope of this project. This may help clarify the kind of_
+_features, contributions, issues or problems the project is looking for._
+
+## Section 2: Relationship with OpenJS Foundation CPC.
+
+_directions: describe how the project intersects with the Cross Project_
+_Council._ 
+
+### 2.1 Other Formal Project Relationships (optional)
+
+_directions: describe any additional affiliations or groups that liaise with_
+_the project in a formal way (such as a W3C Community Group, for example)._ 
+
+## Section 3: Establishment of the ${PROJECT TSC}.
+
+_directions: describe the structure of the group responsible for managing_
+_the project and its respective organization and repositories. If there are_
+_specific rules for membership or participation in the group, list them here or_
+_by reference to a governance.md document._
```

Comment:
I’m not sure that the term “Establishment” can be widely understood. Maybe “Leadership” would be better?

Reply:
What about "Creation" - since this is more about the structure and arguably we'd want the projects to put "Leadership" pieces in their governance docs as much as possible to avoid the issue of them having to update their charter too frequently?

Reply:
How about something more generic like "Project Structure"?

Reply:
I think "Project Leadership" would be a good mix of what has been suggested and I think that is what is covered in the "Etablishment" section in the Node.js TSC charter.

Reply:
On the call yesterday, we discussed using 'Governing Body' which was well received because we already have a definition of that in our charter document. Updating PR with this language.